src/main/java/jp/co/soramitsu/sora/crypto/common/KeyTypeEnum.java

```diff
@@ -0,0 +1,21 @@
+package jp.co.soramitsu.sora.crypto.common;
+
+import jp.co.soramitsu.crypto.ed25519.EdDSAKey;
+import jp.co.soramitsu.crypto.ed25519.EdDSASecurityProvider;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+public enum KeyTypeEnum {
```

Comment:
add annotation @FieldDefaults, in this case you could to remove private keywords